.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
   Exclude:
     - config/initializers/forbidden_yaml.rb
     - app/helpers/dynamic_errors_helper.rb
-    - !ruby/regexp /(vendor|bundle|bin|db|tmp)\/.*/
+    - !ruby/regexp /(vendor|bundle|bin|db|tmp|server)\/.*/
```

Comment:
:thinking: Any idea why this change is needed in here?

Reply:
Rubocop picks up the indexed content files named "rubocop.yml" and tries to parse them, fails, and then breaks rubocop entirely. We have no need to scan the RubygemFs::Local filesystem for rubocop violations anyway.

Reply:
I have seen that in other PR as well. Anyway it seems relevant actually to this PR. Let's keep it. :muscle: